def func():
    print("hey")
    yield 1
    print("before yield 2")
    yield 2
gen=func()
next(gen)


def func():
    x=0
    y=1
    while(True):
        x,y=y,x+y
        yield y

def func_outer():
    def func_inner():
        print("hey inside the inner")
    
    return "abc"

a=func()


x=9
try:
    if(x!=10):
        raise ValueError("value in incorrect for x")
    else:
        print("x is 10")

except ValueError as e:
